pkgs/os-specific/bsd/openbsd/pkgs/source.nix

```diff
@@ -0,0 +1,8 @@
+{ fetchFromGitHub }:
+
+fetchFromGitHub {
+  owner = "openbsd";
+  repo = "src";
+  rev = "19e99d061340840dc52e10ae9c02d78a0a455689";
+  sha256 = "1061vb5m7k7q502bqsfrnnszjv9v66m0ksg5jka5jsdn6qa0fd9n";
+}
```

Comment:
I think we should fetch from the canonical CVS repo, not the GitHub mirror.

Reply:
We're having trouble fetching this, e.g. 

```bash
cvs -d :pserver:anoncvs@anoncvs4.usa.openbsd.org:/cvs checkout -rOPENBSD_7_5 -P src
```
(with a shell with `openssh` `cvs` `inetutils`) is not working.

Reply:
This looks like it's working for me:

```nix
fetchcvs {
  cvsRoot = "anoncvs@anoncvs.fr.openbsd.org/cvs";
  module = "src";
  tag = "OPENBSD_7_5";
  sha256 = lib.fakeSha256;
}
```

In a shell you'd have to set CVS_RSH=ssh to match the behaviour of fetchcvs.

But actually, why can't we just use the source tarball they publish?

https://cdn.openbsd.org/pub/OpenBSD/7.5/src.tar.gz

Reply:
> In a shell you'd have to set CVS_RSH=ssh to match the behaviour of fetchcvs.

oh!

> But actually, why can't we just use the source tarball they publish?

Oh great. I like the look of "CDN". Yes let's do that.

Reply:
Probably wouldn't be too bad using `srcs`, but this is also fine.